test/unit/spec/PollController_spec.js

```diff
@@ -0,0 +1,119 @@
+define([
+  'okta',
+  'q',
+  '@okta/okta-auth-js/jquery',
+  'helpers/mocks/Util',
+  'helpers/dom/PollingForm',
+  'helpers/util/Expect',
+  'LoginRouter',
+  'sandbox',
+  'helpers/xhr/POLLING',
+  'helpers/xhr/CANCEL',
+  'helpers/xhr/SUCCESS'
+],
+function (Okta, Q, OktaAuth, Util, PollingForm, Expect, Router, $sandbox, resPolling, resCancel, resSuccess) {
+
+  var { _, $ } = Okta;
+  var itp = Expect.itp;
+
+  function setup (settings, res) {
+    settings || (settings = {});
+    var successSpy = jasmine.createSpy('successSpy');
+    var setNextResponse = Util.mockAjax();
+    var baseUrl = window.location.origin;
+    var authClient = new OktaAuth({url: baseUrl});
+    var router = new Router(_.extend({
+      el: $sandbox,
+      baseUrl: baseUrl,
+      authClient: authClient,
+      globalSuccessFn: successSpy
+    }, settings));
+    Util.registerRouter(router);
+    Util.mockRouterNavigate(router);
+    Util.mockJqueryCss();
+    setNextResponse(res || [resPolling, resPolling, resPolling]);
+    router.refreshAuthState('polling-token');
+    settings = {
+      router: router,
+      successSpy: successSpy,
+      form: new PollingForm($sandbox),
+      ac: authClient,
+      setNextResponse: setNextResponse
+    };
+    return Expect.waitForPoll(settings);
+  }
+
+  Expect.describe('Polling', function () {
+    describe('PollingForm Content', function () {
+      itp('shows the correct content on load', function () {
+        return setup().then(function (test) {
+          const title = 'There are too many users trying to sign in right now. We will automatically retry in';
```

Comment:
isnt the complete title populated by now with 

          const title = 'There are too many users trying to sign in right now. We will automatically retry in 1 second';